### DISCUSSION TITLE
Add Full Hidden mod a predecessor of Hidden

### DISCUSSION THREAD

**Discussion by xDololow** [ppy/osu#15324] [Ideas]
Basically, it just makes the map invisible. That's it, folks. And obviously, it should not be availble with FL mod (and others that affect AR like traceable).

**Comment by integerrr** [comment]
What do you mean by predecessor? And should HR not be available with this "full hidden" mod then?

**Reply by xDololow** [reply]
> What do you mean by predecessor?

just a fancy word
> And should HR not be available with this "full hidden" mod then?

No, because it affects CS, OD and the map itself. By AR mods I meant traceable and other mods like that
